styles.css

```diff
@@ -84,6 +86,53 @@ article {
   cursor: pointer;
 }
 
+/* Middle Section */
+
+.middle-column {
+  width: 60%;
+}
+
+.bear-box {
+
+}
+
+.bear {
+  height: 500px;
+  margin-top: 170px;
+  margin-left: 80px;
+}
+
+.background {
```

Comment:
1. I don't know if we will want to keep the height at a %. This will force the picture to repeat if the height is left like this. 
2. I think we need to force a minimum width as well so the  bear doesn't slide outside of the box and into the side elements.